ArcGISRuntimeSDKQt_CppSamples/Layers/LoadWfsXmlQuery/LoadWfsXmlQuery.cpp

```diff
@@ -0,0 +1,101 @@
+// [WriteFile Name=LoadWfsXmlQuery, Category=Layers]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "LoadWfsXmlQuery.h"
+
+#include "Map.h"
+#include "MapQuickView.h"
+#include "FeatureLayer.h"
+#include "WfsFeatureTable.h"
+
+using namespace Esri::ArcGISRuntime;
+
+namespace
+{
+  static const QString xmlQueryString = R"(
```

Comment:
An unfortunate C++ issue is that `static` has several meanings in different contexts. In this instance `static` and an anonymous namespace are actually the same thing! You should remove the `static` keyword from here.